### PR TITLE
Bra 15 homework 01

### PR DESCRIPTION
**Pull request by dev-kucha**
Lesson 15
Part 1

### CODE REVIEW

File: common.blocks/blog-page/css/stiles.css [hunk 1]
```diff
@@ -0,0 +1,236 @@
...
+    }
+}
+    
+@media screen and (max-width:768px) {
+    .post-2 {
+        display: none;
+    }
+}
+    
+@media screen and (max-width:320px) {
```
Comment (roskyba) [inline common.blocks/blog-page/css/stiles.css]: Please, repair this `@media` rule here and in all your code

File: index.html [hunk 2]
```diff
@@ -0,0 +1,283 @@
...
+                    <div class="page__title">
+                        Gallery
+                    </div>
+                    <div class="page__subtitle">
+                        Consectetur adipisicing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua.
+                        Ut
+                        enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo
+                        consequat.
+                    </div>
+                    <div class="gallary__inner">
```
Comment (roskyba) [inline index.html]: gallary ==> gallery

**Comment by dev-kucha** [comment]
All found bugs are fixed 22-03-2021 21:03

**Review by roskyba — APPROVED** [review]

**Comment by dev-kucha** [comment]
Done

**Pull request closed without merging**